docs/conversion_examples_gallery/conftest.py

```diff
@@ -23,15 +23,33 @@ def add_data_space(doctest_namespace, tmp_path):
     doctest_namespace["output_folder"] = Path(tmp_path)
 
 
-
 python_version = platform.python_version()
 os = platform.system()
-# Hook to conditionally skip doctests in deeplabcut.rst for Python 3.9 on macOS (Darwin)
+
+# Hook to conditionally skip doctests
 def pytest_runtest_setup(item):
     if isinstance(item, pytest.DoctestItem):
         test_file = Path(item.fspath)
+
         # Check if we are running the doctest from deeplabcut.rst
         if test_file.name == "deeplabcut.rst":
             # Check if Python version is 3.9 and platform is Darwin (macOS)
             if version.parse(python_version) < version.parse("3.10") and os == "Darwin":
                 pytest.skip("Skipping doctests for deeplabcut.rst on Python 3.9 and macOS")
+
+        # Check if we are running the doctest from inscopixsegmentation.rst
+        if test_file.name == "inscopixsegmentation.rst":
+            # Skip on macOS ARM64
+            if os == "Darwin" and platform.machine() == "arm64":
+                pytest.skip(
+                    "The isx package is currently not natively supported on macOS with Apple Silicon. "
+                    "Installation instructions can be found at: "
+                    "https://github.com/inscopix/pyisx?tab=readme-ov-file#install"
+                )
+            # Skip on Python 3.13+
+            if version.parse(python_version) >= version.parse("3.13"):
+                pytest.skip(
+                    "Tests are skipped on Python 3.13 because of incompatibility with the 'isx' module "
+                    "Requires: Python <3.13, >=3.9) "
+                    "See: https://github.com/inscopix/pyisx/issues"
```

Comment:
```suggestion
                    "See: https://github.com/inscopix/pyisx/issues/61"
```

let's cite the specific issue

Reply:
Changed now.